### PR TITLE
EPiServer 7.5/8+ support

### LINKED ISSUE #1: Website crashes after Nuget installation because of wrong binding redirects in the web.config
It works, if you fix `<assemblyBinding>` versions manually or update OpenWaves package to the latest in NuGet. I think it would be better, if it works out of box.

image

### PR DESCRIPTION
**Pull request by whyleee**
- Added EPiServer 7.5/8+ support
- Replaced *.pp initialization code to a public method + readme
- Added VPP config transform (fixes #1)
- Fixed OpenWaves binding redirects 
- Fixed creation of a dir for transformed images

### CODE REVIEW

File: RestImageResize.EpiServer/RestImageResize.EPiServer.nuspec [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+      <dependency id="EPiServer.CMS.Core" version="7.0.6" />
       <dependency id="RestImageResize" version="1.0.3" />
-      <dependency id="OpenWaves.ImageTransformations.EPiServer" version="1.0.85.251"/>
+      <dependency id="OpenWaves" version="1.0.85.259" />
+      <dependency id="OpenWaves.ImageTransformations.EPiServer" version="1.0.85.251" />
     </dependencies>
   </metadata>
   <files>
-    <file src="**.pp" target="content"/>
+    <file src="readme.txt" target="" />
```
Comment (Romanets) [inline RestImageResize.EpiServer/RestImageResize.EPiServer.nuspec]: README.md?

File: RestImageResize.EpiServer/readme.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿RestImageResize for EPiServer
```
Comment (Romanets) [inline RestImageResize.EpiServer/readme.txt]: What for do you need two readme files?

File: RestImageResize.EpiServer/web.config.transform [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (Romanets) [inline RestImageResize.EpiServer/web.config.transform]: Is this file included in package?

File: RestImageResize.EpiServer/RestImageResize.EPiServer.nuspec (hunk 1 above)
Comment (whyleee) [inline RestImageResize.EpiServer/RestImageResize.EPiServer.nuspec]: https://docs.nuget.org/create/creating-and-publishing-a-package#automatically-displaying-a-readme.txt-file-during-package-installation

File: RestImageResize.EpiServer/readme.txt (hunk 2 above)
Comment (whyleee) [inline RestImageResize.EpiServer/readme.txt]: One is repo readme, another one for NuGet package to be displayed after installing it

File: RestImageResize.EpiServer/web.config.transform (hunk 3 above)
Comment (whyleee) [inline RestImageResize.EpiServer/web.config.transform]: yep

File: RestImageResize.EpiServer/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+// General info about the product
+[assembly: AssemblyTitle("RestImageResize.EPiServer")]
+[assembly: AssemblyProduct("RestImageResize")]
+[assembly: AssemblyDescription("Includes required dependences and source code for easily start to use RestImageResize package in EPiServer CMS site.")]
+[assembly: AssemblyCompany("Roman Mironets, Creuna Kharkiv office")]
+[assembly: AssemblyCopyright("Copyright © Roman Mironets 2015")]
+
+// Product version
+[assembly: AssemblyVersion("1.1")]
```
Comment (VladimirLevchuk) [inline RestImageResize.EpiServer/Properties/AssemblyInfo.cs]: Can we keep 3-numbers version?

Comment (whyleee) [inline RestImageResize.EpiServer/Properties/AssemblyInfo.cs]: What's the difference? If you have a third number - just add it

Comment (VladimirLevchuk) [inline RestImageResize.EpiServer/Properties/AssemblyInfo.cs]: we had v1.0.0; it would be nice if you make it 1.1.0 in your PR :)

**Pull request merged**